course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -234,103 +353,72 @@
-                    type: 'alreadyInCourse',
-                    fullName: $scope.getProfileFullName(profile),
-                    memberships: memberResult.data.results,
-                    searchTerm: memberResult.config.searchTerm,
-                };
-                $scope.searchInProgress = false;
-            }
-            else {
-                var filteredResults = $scope.filterSearchResults(
-                                          peopleResult.data.results);
-                if (filteredResults.length == 0) {
-                    // didn't find any people for the search term
-                    $scope.clearMessages();
-                    $scope.addWarning = {
-                        type: 'notFound',
-                        searchTerm: peopleResult.config.searchTerm,
-                    };
-                    $scope.searchInProgress = false;
-                }
-                else if (filteredResults.length == 1) {
-                    $scope.addUserToCourse(peopleResult.config.searchTerm,
-                                           {user_id: filteredResults[0].univ_id,
-                                            role_id: $scope.selectedRole.roleId});
-                }
-                else {
-                    $scope.searchResults = filteredResults;
-                    $scope.searchInProgress = false;
-                }
-            }
+        $scope.handleAjaxErrorResponse = function(r) {
+            // angular promise then() function returns a response object,
+            // unpack for our old-style error handler
+            $scope.handleAjaxError(
+                r.data, r.status, r.headers, r.config, r.statusText);
         };
         $scope.isUnivID = function(searchTerm) {
             var re = /^[A-Za-z0-9]{8}$/;
             return re.test(searchTerm);
         };
-        $scope.lookup = function(searchTerm) {
-            var peopleParams = {page_size: 100};
-            var memberParams = {page_size: 100};
-
-            if ($scope.isUnivID(searchTerm)) {
-                peopleParams.univ_id = searchTerm;
-                memberParams.user_id = searchTerm;
-            } else {
-                peopleParams.email_address = searchTerm;
-                memberParams['profile.email_address'] = searchTerm;
-            }
-
-            // first the general people lookup
-            var peopleURL = djangoUrl.reverse('icommons_rest_api_proxy',
-                                              ['api/course/v2/people/']);
-            var peoplePromise = $http.get(peopleURL, {params: peopleParams,
-                                                      searchTerm: searchTerm})
-                                     .error($scope.handleAjaxError);
-
-            // then the course membership lookup
+        $scope.lookupCourseMembers = function() {
+            // exclude xreg people
+            var getConfig = {
+                params: {limit: 100, '-source': 'xreg_map'}};
```

Comment:
Looks like I missed this, should have moved the `limit` into a `drf` subobject.  Will fix.